client/webserver/site/src/js/markets.ts

```diff
@@ -1529,17 +1529,36 @@ export default class MarketsPage extends BasePage {
     const toAsset = isSell ? quoteAsset : baseAsset
     const fromAsset = isSell ? baseAsset : quoteAsset
 
-    // Set the to and from icons in the fee details pane.
-    for (const icon of Doc.applySelector(page.vDetailPane, '[data-icon]')) {
+    const setIcon = (icon: PageElement) => {
       switch (icon.dataset.icon) {
         case 'from':
-          icon.src = Doc.logoPath(fromAsset.symbol)
+          if (fromAsset.token) {
+            const parentAsset = app().assets[fromAsset.token.parentID]
+            icon.src = Doc.logoPath(parentAsset.symbol)
+          } else {
+            icon.src = Doc.logoPath(fromAsset.symbol)
```

Comment:
This new `setIcon` assumes the icon will only be used to represent fees.  If the order verification form wants to put the icons for the traded assets, this helper won't be of help.  Seems like we need `[data-icon]` for the traded asset and `[data-fee-icon]` for the fee asset icon.  Fine in this PR though.